packages/connect-wallet-modal/package.json

```diff
@@ -33,7 +33,7 @@
   "devDependencies": {
     "@lido-sdk/constants": "^1.6.0",
     "@lido-sdk/react": "^1.18.1",
-    "@reef-knot/web3-react": "*",
+    "@reef-knot/web3-react": "^0.1.0",
```

Comment:
You don't need to specify version for devDependencies

Reply:
Agree, thank you